<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CustomPlotMainWindow</class>
 <widget class="QMainWindow" name="CustomPlotMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>612</width>
    <height>362</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CustomPlotMainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout" stretch="1,0,0,0">
    <item>
     <widget class="QWidget" name="widget_2" native="true"/>
    </item>
    <item>
     <widget class="QLabel" name="label_xy">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_horizontal">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_vertical">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>612</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionFix"/>
   <addaction name="separator"/>
   <addaction name="actionHMea"/>
   <addaction name="actionVMea"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="dockWidget">
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents"/>
  </widget>
  <action name="actionFix">
   <property name="text">
    <string>fix</string>
   </property>
  </action>
  <action name="actionHMea">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>hMea</string>
   </property>
  </action>
  <action name="actionVMea">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>vMea</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
